kibana-reports/public/components/report_definitions/create/create_report_definition.tsx

```diff
@@ -113,17 +174,18 @@ export function CreateReport(props) {
           edit={false}
           reportDefinitionRequest={createReportDefinitionRequest}
           httpClientProps={props['httpClient']}
+          timeRange={timeRange}
         />
         <EuiSpacer />
         <ReportTrigger
           edit={false}
           reportDefinitionRequest={createReportDefinitionRequest}
         />
         <EuiSpacer />
-        <ReportDelivery
+        {/* <ReportDelivery
```

Comment:
I dont think this `ReportDelivery` component should be commented out completely- I think a better workaround would be to exclude the `delivery` params from the request body

Reply:
OK, I'll uncomment this part and add one line to delete delivery field before calling server API